plugins/times-of-day/frontend/public/templates/times-of-day-widget.html

```diff
@@ -13,6 +13,6 @@
     <clyd-primary-legend :apps="data.apps" :show-period="false"></clyd-primary-legend>
 
     <div class="clyd-widget__content">
-        <times-of-day-scatter-chart :series="series" :maxSeriesValue="maxSeriesValue" :show-zoom="false" :show-toggle="false" :showDownload="false" height="auto" skin="full"></times-of-day-scatter-chart>
+        <times-of-day-scatter-chart :series="series" :maxSeriesValue="maxSeriesValue" :show-zoom="false" :show-toggle="false" :showDownload="false" skin="full"></times-of-day-scatter-chart>
```

Comment:
why did you remove height: auto?

Reply:
It seems like we are stretching the charts to parent height only. I will revert it.